fe/fe-core/src/main/java/org/apache/doris/nereids/NereidsPlanner.java

```diff
@@ -581,8 +581,7 @@ public Optional<ResultSet> handleQueryInFe(StatementBase parsedStmt) {
             return Optional.of(resultSet);
         } else if (child instanceof PhysicalEmptyRelation) {
             List<Column> columns = Lists.newArrayList();
-            PhysicalEmptyRelation physicalEmptyRelation = (PhysicalEmptyRelation) physicalPlan.child(0);
-            for (int i = 0; i < physicalEmptyRelation.getProjects().size(); i++) {
+            for (int i = 0; i < physicalPlan.getOutput().size(); i++) {
```

Comment:
what is the relation between this for loop with round function?

Reply:
when we do computeprecision, we do not choose any signature, so its child do not cast to integer type, so child maybe any type

Reply:
this is another bug, and have no relation with round function